core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -388,4 +388,6 @@ private TableProperties() {}
   public static final int ENCRYPTION_DEK_LENGTH_DEFAULT = 16;
 
   public static final int ENCRYPTION_AAD_LENGTH_DEFAULT = 16;
+
+  public static final String ROW_LINEAGE = "row-lineage";
```

Comment:
nit: existing properties have `enabled` for boolean flags. maybe `row-lineage-enabled`?

Reply:
> we defined the field as row-lineage

you meant in the spec? if yes, agree that we should keep them the same.